gpflow/expectations/expectations.py

```diff
@@ -121,7 +121,8 @@ def _init_expectation(
     obj1, feat1 = obj1 if isinstance(obj1, tuple) else (obj1, None)
     obj2, feat2 = obj2 if isinstance(obj2, tuple) else (obj2, None)
     return (
-        cast(ProbabilityDistribution, p),
+        # type-ignore instead of cast, because it dependes on versions whether a cast is necessary.
+        p,  # type: ignore
```

Comment:
Do you not get an unnecessary type ignore error then in one of the versions? Update: just spotted the config change.

Reply:
> Do you not get an unnecessary type ignore error then in one of the versions? Update: just spotted the config change.

:+1:  Yes, there are a couple of places where I've given up on writing code that is type-correct with all versions of numpy.